.github/dependabot.yml

```diff
@@ -1,8 +1,11 @@
 version: 2
 updates:
-- package-ecosystem:
-  - github-actions
-  - maven
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
Do you believe it is necessary to check for new versions of GitHub actions daily?
`weekly` without specifying the time should be good enough for those, I think.
also the time can be omitted

Reply:
> Do you believe it is necessary to check for new versions of GitHub actions daily? `weekly` without specifying the time should be good enough for those, I think. also the time can be omitted

I just used the same settings as for the maven dependencies. Why do we have a time on the daily check anyway?